service/src/main/java/io/camunda/service/AuthorizationServices.java

```diff
@@ -72,35 +59,6 @@ public SearchQueryResult<AuthorizationEntity> search(final AuthorizationQuery qu
                 .searchAuthorizations(query));
   }
 
-  public List<String> getAuthorizedApplications(
-      final Map<EntityType, Set<String>> ownerTypeToOwnerIds) {
-    if (!securityConfiguration.getAuthorizations().isEnabled()) {
-      // if authorizations are not enabled, we default to a wildcard authorization which is
-      // needed for frontend side checks
-      return List.of("*");
-    }
-
-    if (ownerTypeToOwnerIds == null || ownerTypeToOwnerIds.isEmpty()) {
```

Comment:
FYI: This check was missing in the `AuthorizationChecker`. It will be introduced with https://github.com/camunda/camunda/pull/36081